lms/static/karma_lms_coffee.conf.js

```diff
@@ -15,38 +16,32 @@ var options = {
     // Avoid adding files to this list. Use RequireJS.
     libraryFilesToInclude: [
         {pattern: 'xmodule_js/common_static/js/vendor/requirejs/require.js', included: true},
-        {pattern: 'js/RequireJS-namespace-undefine.js', included: true},
-
-        {pattern: 'common/js/vendor/jquery.js', included: true},
-        {pattern: 'common/js/vendor/jquery-migrate.js', included: true},
-        {pattern: 'xmodule_js/common_static/js/vendor/jquery.event.drag-2.2.js', included: true},
-        {pattern: 'xmodule_js/common_static/js/vendor/slick.core.js', included: true},
-        {pattern: 'xmodule_js/common_static/js/vendor/slick.grid.js', included: true},
-
-        {pattern: 'xmodule_js/common_static/js/vendor/requirejs/require.js', included: true},
+        {pattern: 'lms/js/spec/main_requirejs_coffee.js', included: true},
 
+        {pattern: 'js/RequireJS-namespace-undefine.js', included: true},
         {pattern: 'xmodule_js/common_static/coffee/src/ajax_prefix.js', included: true},
-        {pattern: 'common/js/vendor/jquery.js', included: true},
-        {pattern: 'common/js/vendor/jquery-migrate.js', included: true},
         {pattern: 'common/js/vendor/underscore.js', included: true},
         {pattern: 'common/js/xblock/*.js', included: true},
+        {pattern: 'edx-ui-toolkit/js/utils/global-loader.js', included: true},
```

Comment:
For the record, @douglashall and I agreed to use the master versions of both this file and main_requirejs_coffee.js. With those changes, the JS build is now green.